### ISSUE TITLE
[feature request] Reduced Console information 

### ISSUE THREAD

**Issue by ArcticRaven** [Auxilor/EcoEnchants#17]
Hey!

So there's a ton of information that gets spit to the console during startup - the majority of which is simply informational. Would it be possible to implement a config option that reducing this large amount of logging? It makes it difficult when watching live console as it quickly floods what has already been written, and I'm pretty lazy so pulling logs every restart while debugging and error hunting takes a collectively long time. 

TL;DR - option to reduce the amount of informational logging to console

Thank for considering!
